build_package.py

```diff
@@ -314,41 +299,117 @@ def build_win32_installer_package(variant, version):
         make_path(agent_source_root, "DESCRIPTION.rst"),
         convert_path("source_root/DESCRIPTION.rst"),
     )
-    pyinstaller_spec_path = os.path.join(
-        agent_source_root, "win32", "scalyr-agent.spec"
-    )
-
-    shutil.copy(pyinstaller_spec_path, "scalyr-agent.spec")
 
     shutil.copy(
-        os.path.join(agent_source_root, "win32", "dynamic_modules.py"),
-        "dynamic_modules.py",
+        os.path.join(agent_source_root, "win32", "scalyr_agent.wxs"), "scalyr_agent.wxs"
     )
 
-    shutil.copy(
-        os.path.join(agent_source_root, "win32", "wix-heat-bin-transform.xsl"),
-        "wix-heat-bin-transform.xsl",
-    )
+    agent_package_path = os.path.join(agent_source_root, "scalyr_agent")
+
+    add_data = {
+        os.path.join("data_files", "install_info"): "scalyr_agent"
+    }
+
+    # Add monitor modules as hidden imports, since they are not directly imported in the agent's code.
+    hidden_imports = [
+        "scalyr_agent.builtin_monitors.apache_monitor",
+        "scalyr_agent.builtin_monitors.graphite_monitor",
+        "scalyr_agent.builtin_monitors.mysql_monitor",
+        "scalyr_agent.builtin_monitors.nginx_monitor",
+        "scalyr_agent.builtin_monitors.shell_monitor",
+        "scalyr_agent.builtin_monitors.syslog_monitor",
+        "scalyr_agent.builtin_monitors.test_monitor",
+        "scalyr_agent.builtin_monitors.url_monitor",
+        "scalyr_agent.builtin_monitors.windows_event_log_monitor",
+        "scalyr_agent.builtin_monitors.windows_system_metrics",
+        "scalyr_agent.builtin_monitors.windows_process_metrics",
+        "scalyr_agent.builtin_monitors.openmetrics_monitor",
```

Comment:
May also need to add kubernetes_openmetrics_monitor module which was added recently. But that monitor is kinda specific to Kubernetes version of the agent so one option is to exclude it from the Windows bundle.